prov/verbs/src/verbs_cq.c

```diff
@@ -189,9 +188,7 @@ vrb_poll_events(struct vrb_cq *_cq, int timeout)
 		rc--;
 	}
 	if (fds[1].revents & POLLIN) {
-		do {
-			ret = read(fds[1].fd, &data, 1);
-		} while (ret > 0);
+        fd_signal_reset(&_cq->signal);
```

Comment:
indentation is off

Reply:
We use tabs.  In general we follow the Linux kernel coding style because several of us need to work in that development area as well.  https://www.kernel.org/doc/html/v4.10/process/coding-style.html